test/pageobjects/overveiw.page.js

```diff
@@ -0,0 +1,98 @@
+import Page from './page.js';
+import { getNumberFromString } from '../../utils/regexToNumber.js';
+
+class OverwiewPage extends Page {
+
+    get item() {
+        return $('div[data-test="inventory-item-name"]');
+    }
+
+    get allPrices() {
+        return $$('div[data-test="inventory-item-price"]');
+    }
+
+    get subtotal() {
+        return $('div[data-test="subtotal-label"]');
+    }
+
+    get tax() {
+        return $('div[data-test="tax-label"]');
+    }
+
+    get total() {
+        return $('div[data-test="total-label"]');
+    }
+
+    get finishBtn() {
+        return $('button[data-test="finish"]');
+    }
+
+    get cancelBtn() {
+        return $('button[data-test="cancel"]');
+    }
+
+    get continueShopping() {
+        return $('button[data-test="continue-shopping"]');
+    }
+
+    async clickContinueShopping() {
+        await this.continueShopping.click();
+    }
+
+    async clickItem() {
+        await this.item.click();
+    }
+
+    async clickFinishBtn() {
+        await this.finishBtn.click();
+    }
+
+    async clickCancelBtn() {
+        await this.cancelBtn.click();
+    }
+
+    async calculateTotal() {
```

Comment:
@karenn32 For such complicated methods the JSDoc/TSDoc is required. It reduces the required time for maintenance (fixes) for the tests and review process.